examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability-constant-sharpening.prm

```diff
@@ -107,8 +113,14 @@ subsection VOF
   subsection mass conservation
     set monitoring      = true
     set monitored fluid = fluid 1
-    set tolerance       = 1e-6
-    set verbosity       = quiet
+    set tolerance       = 1e-2
```

Comment:
Should be lowered here too?

Reply:
Actually here it should be completely removed. Because constant sharpening has no tolerance.